diff --git a/dist/sandbox/sandbox-manager.js b/dist/sandbox/sandbox-manager.js
index 3a2d9c8abd01fefff36edea0cd76e9c31a3ed411..7e422b7a159da0ae00902cefcd5d38ac501c223c 100644
--- a/dist/sandbox/sandbox-manager.js
+++ b/dist/sandbox/sandbox-manager.js
@@ -1293,7 +1293,7 @@ async function wrapWithSandbox(command, binShell, customConfig, abortSignal, opt
                 unsetEnvVars: credentialRestrictions.unsetEnvVars,
                 setEnvVars: credentialRestrictions.setEnvVars,
                 maskedFileBinds: credentialRestrictions.maskedFileBinds,
-                allowUnixSockets: getAllowUnixSockets(),
+                allowUnixSockets: customConfig?.network?.allowUnixSockets ?? getAllowUnixSockets(),
                 allowAllUnixSockets: getAllowAllUnixSockets(),
                 allowLocalBinding: getAllowLocalBinding(),
                 allowMachLookup: getAllowMachLookup(),
